### DISCUSSION TITLE
Problem to sing-up

### DISCUSSION THREAD

**Discussion by SanchezAle** [denoland/deno#15303] [Q&A] [answered]
Hi guys,

I have a problem when i sign-in with my github account at deno deploy

{
  "code": "gitHubAccountBlocked",
  "message": "Your account has been temporarily blocked. Please try again later, or use a different account."
}

This error appeared today 07-24-2022

I don't understand because my account is not blocked. Someone will can help me? what is the reasson?

Thanks.

**Comment by kitsonk** [comment, marked as answer]
Support issues for Deno Deploy should be directed to deploy@deno.com.

**Reply by SanchezAle** [reply]
Thanks!

**Reply by zillTheTerminator** [reply]
@SanchezAle Same issue here, do you got any solution or feedback?

**Reply by SanchezAle** [reply]
This was support's response:

We filter users because new accounts are often spammers. Try again in a week.

**Comment by dnk8n** [comment]
Not a great user experience when I am attempting to introduce a class full of students to both github and deno at the same time. Very frustrating!

**Comment by lvlasoud8** [comment]
also same issue!

**Comment by igorrake** [comment]
Wow! There must be a better way to avoid spammers.
